README.md

```diff
@@ -60,6 +60,32 @@ docker-compose run app bandit -r . -x manage.py,docker_entrypoint.py
 docker-compose run app flake8
 ```
 
+### Dependency management
```

Comment:
@Jkrzy this is great! can you remove this from the main readme and add it to a single file within `/docs`. Perhaps something like `/docs/dependency-management.md`. I'll take care of adding the link in the `/docs/README.md`. Thanks! See #752 for more info on `/docs`.